network/peer/peer.go

```diff
@@ -755,7 +756,7 @@ func (p *peer) handle(msg message.InboundMessage) {
 		msg.OnFinishedHandling()
 		return
 	case *p2p.PeerList:
-		p.handlePeerList(m)
+		p.handlePeerList(ctx, m)
```

Comment:
Should we add a context to all of the handler functions rather than just for peer list?

Reply:
> Should we add a context to all of the handler functions rather than just for peer list?

I don't think we should pre-emptively add a context parameter to the other functions because it would be unused

> I assume we only added it here because it was the only one that passed down the context another level

That is correct